package.json

```diff
@@ -24,14 +24,14 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "ember-cli-babel": "^6.0.0",
+    "ember-cli-dependency-checker": "^2.0.0",
```

Comment:
This is generally only a devDep, how are we using this directly? 

Did you mean ember-cli-version-checker?